json-endpoints/class.wpcom-json-api-get-site-endpoint.php

```diff
@@ -315,6 +315,8 @@ public function build_current_site_response( ) {
 					$response['options']['is_multi_network'] = (bool) get_option( 'jetpack_is_main_network', true  );
 					$response['options']['is_multi_site'] = (bool) get_option( 'jetpack_is_multi_site', true );
 
+					// Show available updates.
+					$response['options']['updates'] = Jetpack_Options::get_option( 'updates' );
```

Comment:
I think this part will change. I would better to omit it from and then use what ever is coming from .com instead.